spec/Section 3 -- Type System.md

```diff
@@ -837,7 +837,6 @@ input ExampleInputObject {
 Original Value          | Variables       | Coerced Value
 ----------------------- | --------------- | -----------------------------------
 `{ a: "abc", b: 123 }`  | {null}          | `{ a: "abc", b: 123 }`
-`{ a: 123, b: "123" }`  | {null}          | `{ a: "123", b: 123 }`
```

Comment:
well .... the error is an error saying that the type is wrong. I don't think this has anything to do with input coercion. That's the reason I decided to remove it.